printer_mods/42bios/LED_mount/README.md

```diff
@@ -0,0 +1,6 @@
+Simple LED-mount for [Lumentech LT3 profiles.](https://lumentec.eu/Aluminium-Eckig-LED-Profil-100cm-45-fuer-8-12mm-LED-Streifen-Profil-LT3-Abdeckung)<br>
+Mount with two M3x10 screws and T- or hammerhead nuts to normal 2020 extrusions.<br>
+Works with 8-12mm LED stripes<br>
+
+version: 0.1<br>
+![GitHub Logo](https://i.imgur.com/oPmW2XH.png)![GitHub Logo](https://i.imgur.com/oP0wyu7.png)
```

Comment:
Can you add the images to your PR instead of referencing them on imgur?  That way the mod is self-contained here and doesn't depend on external services.  Also change the alt text from `GitHub Logo` to something that describes each image.

Reply:
Yes for sure, its my first PR! :) Thank you for feedback

Reply:
I hope its okay now?